### DISCUSSION TITLE
Merging object issue

### DISCUSSION THREAD

**Discussion by tingxie2020** [stuart-lab/signac#959] [Q&A] [answered]
(https://satijalab.org/signac/articles/merging.html)
Signac->Merging object->Create a common peak set
how to start with raw_peak_bc_matrix.h5 files?

if follow the" Analyzing PBMC scATAC-seq" (vignettehttps://satijalab.org/signac/articles/pbmc_vignette.html), 
"Computing QC Metrics" needs to be performed before or after the merging object step?

I'm a beginner with ScATAC-seq analysis so please forgive my naivety.
Thanks

**Comment by timoast** [comment, marked as answer]
> how to start with raw_peak_bc_matrix.h5 files?

The raw peak matrix contains information for every sequenced barcode. Starting from this, the workflow is the same as for the filtered count matrix except that you'd want to remove barcodes containing very low counts. You can do this in `CreateChromatinAssay()` by setting the `min.features` parameter.

> "Computing QC Metrics" needs to be performed before or after the merging object step?

You can compute QC metrics before or after merging
